lib/bgs/form686c.rb

```diff
@@ -96,8 +97,26 @@ def get_state_type(payload)
       'Started'
     end
 
+    # if we are setting the claim to be manually reviewed
+    # and the Veteran is currently receiving pension benefits
+    # set the claim type code to 130DAEBNPMCR (PMC eBenefits Dependency Adjustment Reject)
+    # else use 130DPNEBNADJ (eBenefits Dependency Adjustment)
+    def get_claim_type(proc_state)
+      if proc_state == 'MANUAL_VAGOV'
```

Comment:
Although this may be a widely used pattern in the repo (and outside the scope of this PR), should this string be a constant somewhere? Something like:
```suggestion
      if proc_state == BGS::ProcStates::MANUAL_VAGOV
```

Reply:
> Although this may be a widely used pattern in the repo (and outside the scope of this PR), should this string be a constant somewhere? Something like:

There are only 3 proc states ('Started', 'Ready', 'MANUAL_VAGOV') so not sure we need this but where would you suggest we add it?

Reply:
So I typed out a reply, but then hit resolve conversation too quickly, thinking it would comment and resolve, but it wiped my comment. 

Basically I was saying I got a little carried away. Someone I respected at my old job put a widely used string in an Enum, so i thought I'd try to apply here.

I want to try and not repeat history where I "fix" things that aren't broken. We can always refactor when we have the time/need. Sorry for the noise here :)